java/src/main/java/com/google/oak/client/weather_lookup_client/Main.java

```diff
@@ -29,6 +30,7 @@
 import io.grpc.ManagedChannelBuilder;
 import java.net.URL;
 import java.nio.charset.StandardCharsets;
+import java.time.Clock;
```

Comment:
Does Android use this clock as well?

Reply:
Yup, it does: https://developer.android.com/reference/java/time/Clock